layers/+distribution/spacemacs/packages.el

```diff
@@ -369,6 +369,7 @@
          :exit t)
         ("n" spacemacs/quick-ahs-forward)
         ("N" spacemacs/quick-ahs-backward)
+        ("p" spacemacs/quick-ahs-backward)
         ("R" ahs-back-to-start)
```

Comment:
So that it's convenient for both Emacs and Vim users. We do it in a few other places, too.